helmfile.d/values/fluentd/aggregator-common.yaml.gotmpl

```
@@ -16,11 +16,27 @@ extraDeploy:
   - apiVersion: v1
     kind: Secret
     metadata:
-      name: {{ .Release.Name }}-s3-credentials
+      name: {{ .Release.Name }}-storage-credentials
     type: Opaque
     stringData:
+      {{- if eq .Values.objectStorage.type "s3" }}
       AWS_ACCESS_KEY_ID: {{ .Values.objectStorage.s3.accessKey }}
       AWS_ACCESS_SECRET_KEY: {{ .Values.objectStorage.s3.secretKey }}
+      {{- else if eq .Values.objectStorage.type "azure" }}
+      STORAGE_ACCOUNT: {{ .Values.objectStorage.azure.storageAccountName }}
+      STORAGE_ACCESS_KEY: {{ .Values.objectStorage.azure.storageAccountKey }}
+      {{- end }}
+
+
+image:
+  registry: ghcr.io
+  repository: elastisys/fluentd
+  tag: v5.19.0-ck8s1
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: https://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ## TDOD: reset to main image when it's updated
```

So we find this when grepping for technical debt! :smile: 
```suggestion
  ## TODO: reset to main image when it's updated
```